src/reference_transmogrifier/models/inspector/extra_hardware.py

```diff
@@ -98,25 +120,26 @@ def cache_per_core(self) -> Self:
         keys = ["l1d cache", "l1i cache", "l2 cache", "l3 cache"]
         for k in keys:
             value = self.get(k)
-            if not value:
-                raise ValueError(f"{k} not found in input")
+            if not value or not isinstance(value, str):
+                self[k] = None
+                continue
 
             try:
                 size_str = value.split("(")[0].strip()
                 instances_str = value.split("(")[1].strip().split(" ")[0]
-            except (IndexError, ValueError):
-                raise ValueError(f"{k} has malformed input")
+                num_instances = int(instances_str)
+                total_size_bytes = ByteSize._validate(size_str, None)
+                per_core_bytes = total_size_bytes // num_instances
+                self[k] = per_core_bytes
+            except Exception:
+                self[k] = None
 
-            num_instances = int(instances_str)
-            total_size_bytes = ByteSize._validate(size_str, None)
-            per_core_bytes = total_size_bytes // num_instances
-            self[k] = per_core_bytes
         return self
 
 
 class CPU(BaseModel):
     physical: CPUSummary
-    physical_0: PhysicalCPU
+    physical_0: Optional[PhysicalCPU] = None
```

Comment:
After "evicting" the NU and NCAR nodes, is this line still required?

Reply:
Those don't appear to be